### DISCUSSION TITLE
What is the best wayto set conditional entries to `app.head` in Nuxt config?

### DISCUSSION THREAD

**Discussion by stefanobartoletti** [nuxt/nuxt#26788] [Questions] [answered]
I'm trying to find out what is the best way to include conditional entries into 

```js
export default defineNuxtConfig({
  app: {
    head: {
        // entries to be put here
    }
  }
})
```

My use case are site analytics scripts that are not needed during development, and that generate a lot of noisy console warnings in this environment.

I am trying something like this

```js
  app: {
    head: {
      meta: [
        process.env.GOOGLE_VERIFICATION ? { name: 'google-site-verification', content: process.env.GOOGLE_VERIFICATION } : false,
      ],
      script: [
        // PostHog
        process.env.PH_API_KEY ? { children: `analytics script requiring '${process.env.PH_API_KEY}` } : false,
      ],
    },
  },
```

This seems to work nicely, it only loads these things if the relevant environment variables are set, which I only do in production.

The reason I'm asking is because I don't seem to find references to conditionally loading stuff here in the Nuxt docs, and I'd like a confirmation that this is a good approach or if there are some other best practices about this.

Thanks :)

**Comment by dsvgl** [comment, marked as answer]
https://nuxt.com/docs/getting-started/configuration#environment-overrides ;)

**Reply by stefanobartoletti** [reply]
Thanks!
